### PR TITLE
Add .github/workflows/workflow.yml

### PR DESCRIPTION
**Pull request by abetomo**

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Node CI
+on: [push]
```
Comment (kou) [inline .github/workflows/workflow.yml]: `[push, pull_request]` will be better.

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@master
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Groonga
+      run: |
+        set ARCHIVE=groonga-9.0.3-65dc3cd-x64.zip
+
+        choco install -y curl 7zip.commandline
+        curl -O https://packages.groonga.org/tmp/%ARCHIVE%
```
Comment (kou) [inline .github/workflows/workflow.yml]: Released version at https://github.com/groonga/groonga/releases will be better.
e.g. https://github.com/groonga/groonga/releases/download/v9.0.8/groonga-9.0.8-x64.zip built by MinGW or
https://github.com/groonga/groonga/releases/download/v9.0.8/groonga-9.0.8-x64-vs2017.zip built by Visual Studio.

Comment (abetomo) [reply .github/workflows/workflow.yml]: I got it.

**Comment by abetomo** [comment]
@kou Thank you for the review.

**Pull request merged**